integration/vscode/ada/package.json

```diff
@@ -99,6 +99,23 @@
                     }
                 }
             }
+        ],
+        "problemMatchers": [
+            {
+                "name": "ada",
+                "label": "Gnat Ada Problem Matcher",
+                "fileLocation": ["relative", "${workspaceRoot}"],
+                "pattern": [
+                    {
+                        "regexp": "^(.*):(\\d+):(\\d+):\\s+(warning|)(?:\\s+)?(.*)$",
```

Comment:
I'm afraid first part of the regexp won't work on Windows. (Windows has drive letters, like `C:\`). I propose replace `^(.*)` with something like this `^([^:]:?[^:]*)`

What do you think?

Reply:
Hello reznikmm!
I'm fine with your proposal!
So here would be the final regex (i've tested with specific cases and noticed a little typo with the final ':' that was included into the message part):

"^([^:]:?[^:]*):(\\d+):(\\d+):\\s+(warning|)(?:[:]\\s+)?(.*)$"